app/controllers/api/episodes_controller.rb

```diff
@@ -3,4 +3,8 @@ class Api::EpisodesController < Api::BaseController
   represent_with Api::EpisodeRepresenter
   filter_resources_by :podcast_id
   find_method :find_by_guid
+
+  def scoped(relation)
+    relation.published.released
```

Comment:
This allows this API endpoint to be public and to mirror the feed in what episodes it returns.
I have another PR I started where non-public episodes can be returned via an authenticated request, like cms